program/src/processor.rs

```diff
@@ -10,13 +10,14 @@ use {
     solana_account_info::{next_account_info, AccountInfo},
     solana_cpi::{invoke, invoke_signed},
     solana_msg::msg,
-    solana_program::{system_instruction, sysvar::Sysvar},
     solana_program_error::{ProgramError, ProgramResult},
     solana_program_pack::Pack,
     solana_pubkey::Pubkey,
     solana_rent::Rent,
+    solana_system_interface::instruction::{allocate, assign},
     spl_token_2022::{
         extension::PodStateWithExtensions, instruction::initialize_mint2, pod::PodMint,
+        solana_program::sysvar::Sysvar,
```

Comment:
Given https://github.com/solana-program/token-wrap/pull/6#discussion_r1944507305 a work around is importing Sysvar from a token program. That was `solana-program` can be booted from cargo.toml. But perhaps that is a hacky workaround 🤷

Reply:
Haha very clever! To be honest, we'll likely remove the re-export from token-2022 with a breaking change once we update it to use the component crates, so best to just use `solana_program` directly